packages/panels/src/Resources/Pages/CreateRecord.php

```diff
@@ -183,7 +184,14 @@ protected function handleRecordCreation(array $data): Model
 
     protected function associateRecordWithTenant(Model $record, Model $tenant): Model
     {
-        return static::getResource()::getTenantRelationship($tenant)->save($record);
+        $relationship = static::getResource()::getTenantRelationship($tenant);
+
+        if (!$relationship instanceof HasManyThrough) {
+            return $relationship->save($record);
+        }
+        
+        $record->save();
+        return $record;
```

Comment:
Would it be better to:

```suggestion
        $relationship = static::getResource()::getTenantRelationship($tenant);

        if ($relationship instanceof HasManyThrough) {
            return $record->save();
        }

        return $relationship->save($record);
```
Switching up the conditions makes it easier to read.

Or with a short-hand

```suggestion
        $relationship = static::getResource()::getTenantRelationship($tenant);
        
        return ($relationship instanceof HasManyThrough) ? $record->save() : $relationship->save($record);
```

To Lower the lines of code and to improve readability.

Reply:
@Daveyvdweide - thanks for having a look. I appreciate your suggestions. And agreed, inverting the conditionals improves readability. I pushed up a change that incorporates that. Looking through the rest of the codebase, it seems that traditional if/else conditionals are typically used in place of ternary operators, so I stuck with that to best align with the rest of the code.

I wasn't able to directly commit your proposed change however, because your code was setup to return `$record->save()` (which will return a boolean) rather than `$record` (an object) in the case where `$relationship` is an instance of `HasManyThrough`.

Thanks again for the review - much appreciated!